Using-Docker.md

```diff
@@ -0,0 +1,41 @@
+# Using Docker
+Using docker as the runtime environment for testing and validating new developments is now available using the next commands:
+
+## Build the plugin
+First the plugin must be build using the `yarn build` command.
+
+Once this build was completed - the expected zip location of the plugin is `./build/observabilityDashboards-?.?.?.zip` where as the `?.?.?` represents the version of this dashboard plugin.
+
+> Note that the plugin version must correspond to the OpenSearch-Dashboards version - this information appears [here](opensearch_dashboards.json)
+> 
+
+Once the build is completed, make sure to overide the [Dockerfile](Dockerfile) target zip file with the exact name 
+```
+# Install updated plugin
+RUN /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin install file:///tmp/observabilityDashboards-3.0.0.zip
+```
+
+## Docker imageVersion
+The docker images used by this file are all referencing the [.env](.env) environment file that contains the version field an user/password field that needs to be changed to match your own system. 
```

Comment:
minor typos
```suggestion
The docker images used by this file are all referencing the [.env](.env) environment file that contains the version and user/password fields that need to be changed to match your own system. 
```

> that need to be changed to match your own system

in this context, the environment inside docker container should not need to be affected to user's system?

i'm not sure how to phrase it, maybe use `configuration` instead of `system?